src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -644,4 +644,12 @@ if(NOT ENABLE_${upper_LAI})
 endif()
 option(GEOSX_LA_INTERFACE_${upper_LAI} "${upper_LAI} LA interface is selected" ON)
 
+###############################
+# NvToolExt
+###############################
+if (ENABLE_CUDA_NVTOOLSEXT)
```

Comment:
Can you also check that `ENABLE_CUDA` is `ON?

Reply:
Ok i'll do so.